index.php

```diff
@@ -1,7 +1,10 @@
 <?php
 require 'dbconn.php';
```

Comment:
In dbconn.php:
- Probably better to write functionally than procedurally - Ie make a getDB() or something like that to call in your index.php or anywhere else
- May want to move the SQL query into a function within functions.php so it is more viewable/changeable. Eg: function getSongs(DBO $db) : array {}